.circleci/config.yml

```diff
@@ -1,123 +1,50 @@
+          filters: *tags_and_branches
 
-      # This step is executed only on tag release. Please note that currently CircleCI is not able to execute
-      # the integration-tests workflow on tag release. So dev-promote-prod-from-git-tag job is not in the same
-      # workflow as the integration tests.
-      - orb-tools/dev-promote-prod-from-git-tag:
+      - orb-tools/publish:
+          name: publish-dev
           orb-name: financial-times/golang-ci
+          pub-type: dev
+          vcs-type: << pipeline.project.type >>
           context: cm-team-orb-publishing
-          add-pr-comment: false
-          perform-branch-check: true
+          enable-pr-comment: false
           requires:
-            - orb-tools/publish-dev
-          filters: *only_tags
-
-  # The integration tests should be run in a separate workflow triggered by the one above so that they use
-  # the latest dev:alpha version of the orb that was released by the publish-dev job above.
-  integration-tests:
-    when: << pipeline.parameters.run-integration-tests >>
-    jobs:
-      - golang-ci/build-and-test:
-          name: integration-tests-default
-          filters: *only_branches
-
-      - golang-ci/build-and-test:
-          name: integration-tests-with-neo4j
-          executor-name: golang-ci/default-with-neo4j
-          filters: *only_branches
-
-      - golang-ci/build-and-test:
-          name: integration-tests-with-neo4j-cluster
-          executor-name: golang-ci/default-with-neo4j-cluster
-          filters: *only_branches
-
-      - golang-ci/build-and-test:
-          name: integration-tests-with-elasticsearch
-          executor-name: golang-ci/default-with-elasticsearch
-          filters: *only_branches
-
-      - golang-ci/build-and-test:
-          name: integration-tests-with-mysql
-          executor-name: golang-ci/default-with-mysql
-          filters: *only_branches
-
-      - golang-ci/build-and-test:
-          name: integration-tests-with-postgres
-          executor-name: golang-ci/default-with-postgres
-          filters: *only_branches
-
-      - golang-ci/docker-build:
-          name: docker-job-integration-tests
-          filters: *only_branches
+            - orb-tools/lint
+            - orb-tools/pack
+            - orb-tools/review
+            - shellcheck/check
+          filters: *tags_and_branches
 
-  # Republish the dev:alpha orb every two months to ensure new pipelines don't get rejected due to expired dev orbs.
-  # We need to refer to the dev release of the orb for the integration-tests workflow. However when there is no dev orb
-  # release (or it's expired) this config file is marked as invalid and none of its workflows is allowed to run.
-  keep-dev-orb-alive:
-    triggers:
-      - schedule:
-          cron: "0 0 1 1,3,5,7,9,11 *"
-          filters: *only_master
-    jobs:
-      - orb-tools/pack
-      - orb-tools/publish-dev:
-          orb-name: financial-times/golang-ci
+      - orb-tools/continue:
```

Comment:
**question:** Is this responsible for passing the execution to `test-deploy.yaml` ?

Reply:
Yes, that's right. As far as I remember the other workflows after the `setup` workflow will run in parallel.